install_config/master_node_configuration.adoc

```diff
@@ -344,7 +344,9 @@ permissions.
 |Sets the hostname for the default internal image
 registry. The value must be in `*hostname[:port]*` format.
 For backward compatibility, users can still use `*OPENSHIFT_DEFAULT_REGISTRY*`
-environment variable but this setting overrides the environment variable.
+environment variable but this setting overrides the environment variable.  When
+this is set, the internal registry must its hostname set as well.  
```

Comment:
s/must its hostname/must set its hostname?

Reply:
thanks, "must have its hostname set as well" is what i think intended. fix pushed.